course_info/static/course_info/js/controllers/DetailsController.js

```diff
@@ -0,0 +1,107 @@
+// TODO - this is pretty much a copy of the people controller with some mods.
+// TODO - this still needs a lot of clean up, we don't need everything in here
+
+(function () {
+    var app = angular.module('CourseInfo');
+    app.controller('DetailsController', DetailsController);
+
+    function DetailsController($scope, $routeParams, courseInstances, $compile,
+                               djangoUrl, $http, $q, $log, $uibModal, $sce) {
+
+        dc = this;
```

Comment:
Can we scope `dc` to the controller construction function by making this
```js
var dc = this;
```
Otherwise, we're making it a global, which I don't think we should be doing.